x/uibc/uics20/memo_handler.go

```diff
@@ -58,13 +58,20 @@ func (mh *MemoHandler) onRecvPacketPrepare(
 		return nil, sdkerrors.Wrap(err, "can't parse ftData.Receiver bech32 address")
 	}
 
+	var memoPayload []byte
 	if strings.EqualFold(ftData.Sender, gmp.DefaultGMPAddress) {
 		events = append(events, "axelar GMP transaction")
 		mh.isGMP = true
-		return events, nil
+		gh := gmp.NewHandler()
```

Comment:
Now gmp incoming payload will be assigned to memoPayload because it is same as normal ibc memo , so we are not doing any separate execution for gmp handler it is same as ibc memo handler